### PR TITLE
Updates docs generation template

### PR DESCRIPTION
**Pull request by flovilmart**
before | after
---|---
capture d ecran le 2018-08-10 a 14 28 49|capture d ecran le 2018-08-10 a 14 28 34

### CODE REVIEW

**Comment by dplewis** [comment]
Is there a specific reason why you changed the template?

On mobile they both look the same. On desktop the Before has better padding. After has better font.

**Comment by flovilmart** [comment]
The template has a fix for the generation for Parse server :) also it’s a fork on our repo so we can gradually make it look better for all docs

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "xmlhttprequest": "1.8.0"
   },
   "devDependencies": {
+    "@parse/minami": "^1.0.0",
```
Comment (dplewis) [inline package.json]: Ok I was wondering where this came from, I didn't see  the fork.

Should this build against the master branch?

Comment (flovilmart) [reply package.json]: https://github.com/nijikokun/minami/compare/master...parse-community:master

it's published already as an Npm package so we're good.

**Pull request merged**